nilearn/experimental/surface/_datasets.py

```diff
@@ -41,8 +42,8 @@ def fetch_nki(n_subjects=1) -> Sequence[SurfaceImage]:
         img = SurfaceImage(
             mesh=fsaverage["pial"],
             data={
-                "left_hemisphere": left_data,
```

Comment:
sticking to left and right to match vali hemipsheres names we have in the plotting modules:

https://github.com/nilearn/nilearn/blob/1c90f3fc379d6450b2f4493001b8f9747366276c/nilearn/plotting/surf_plotting.py#L29

Can be changed but would be nice to keep things internally consistent